varnish/content.md

```diff
@@ -70,10 +87,10 @@ $ docker run %%IMAGE%% varnishd -x parameter
 $ docker run %%IMAGE%% varnishd -a :8080 -b 127.0.0.1:8181 -t 600 -p feature=+http2
 ```
 
-### Exposing the port
+## Exposing the port
 
 ```console
 +$ docker run --name my-running-varnish --tmpfs /var/lib/varnish:exec -d -p 8080:80 my-varnish
 ```
 
-Then you can hit `http://localhost:8080` or `http://host-ip:8080` in your browser.
+	Then you can hit `http://localhost:8080` or `http://host-ip:8080` in your browser.
```

Comment:
This seems odd; did you mean to put this in a code block?

Reply:
Uh, i didn't